[Android][iOS] fix LocalNotification Android plugin to work with all Phonegap versions

grnadav opened the pull request:
Cordova ref was outdated. updated for backwards comparability.
Because of wrongly handling github pull-request, I accidently added a second pull request here: in iOS and Android Screenshot plugin, added a save screenshot as file method that returns base64 presentation of the taken screenshot

grnadav committed:
fix LocalNotification Android plugin to work with all Phonegap versions (up to 2.0)

imagerio committed:
Merge remote-tracking branch 'upstream/master'

imagerio committed:
added save screenshot to file as well as to base64. also added base 64 lib

zkhalapyan commented:
We are having an issue with Android 4.1 and LocalNotification plugin where all the reminders disappear when the device is shutdown or the process is closed. @grnadav Would you know if this commit has fixed this issue?

grnadav committed:
added saveScreenshotAsFile functionality to Screenshot plugin

added saveScreenshotAsFile functionality to Screenshot plugin (Android's
turn) so it returns base64 presentation of the taken screenshot

grnadav committed:
Merge remote-tracking branch 'upstream/master'

grnadav commented:
@zkhalapyan I don't think this is related. It is just a fix that makes LocalNotification plugin work with all phonegap versions, so if it already works with your version than this won't (shouldn't) affect you
